init/kubic-init.systemd.conf

```
@@ -15,6 +18,11 @@ SyslogIdentifier=kubic-init
 # TODO: see https://www.freedesktop.org/software/systemd/man/systemd.unit.html#OnFailure=
 # OnFailure=kubic-init-reset.service
 
+# Define configuration files and don't start if they don't exist
+Environment="KUBE_CFG=/etc/kubernetes/admin.conf"
```

I would call this `KUBECONFIG`, as `kubectl` recognizes this env var and we would not need the following `--kubeconfig=${KUBE_CFG}` arguments...

```suggestion
Environment="KUBECONFIG=/etc/kubernetes/admin.conf"
```